### PR TITLE
Added In Assignment 05 & 06

### PR DESCRIPTION
**Pull request by Ashwin2812**

### CODE REVIEW

**Review by shubhamdalal49 — APPROVED** [review]

File: Ashwin/JavaScript/09_Assignment/09_Assignment_3.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            remainder = temp % 10;
+            reverse = reverse * 10 + remainder;
+            temp = parseInt(temp / 10);
+          }
+          if (reverse == number) return true;
+        }
+      }
+    </script>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_3.html]: make correct indentation 
add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Ashwin/JavaScript/09_Assignment/09_Assignment_1.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+        <style>
+          .box {
+            height: 250px;
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_1.html]: proper indent

File: Ashwin/JavaScript/09_Assignment/09_Assignment_2.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+        <style>
+          .box {
+            height: 250px;
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_2.html]: proper indent

File: Ashwin/JavaScript/09_Assignment/09_Assignment_4.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+        <style>
+          .box {
+            height: 250px;
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_4.html]: proper indent

Comment (nileshp1122) [reply Ashwin/JavaScript/09_Assignment/09_Assignment_4.html]: 4 line space

File: Ashwin/JavaScript/09_Assignment/09_Assignment_4.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                    <h1>String Palindrome</h1>
+                    <input type="text" id="no" class="form-control" />
+                    <button onclick="palindromeString()" class="form-control btn-danger mt-3">Calculate</button>
+                    <p id="p" class="mt-3" style="color: blueviolet; font-size: 35px"></p>
+                </div>
+            </div>
+            <div class="col-md-4"></div>
+        </div>
+        <script>
+            function palindromeString() {
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_4.html:36]: proper indent all JS code

File: Ashwin/JavaScript/09_Assignment/09_Assignment_5.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+        <style>
+          .box {
+            height: 250px;
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_5.html]: proper indent

File: Ashwin/JavaScript/09_Assignment/09_Assignment_5.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+                    <h1>Double letters in word</h1>
+                    <input type="text" id="no" class="form-control"/>
+                    <button onclick="doubleLetters()" class="form-control btn-danger mt-3">Calculate</button>
+                    <p id="p" class="mt-3" style="color: blueviolet; font-size: 35px;"></p>
+                </div>
+            </div>
+            <div class="col-md-4"></div>
+        </div>
+        <script>
+            function doubleLetters() {
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_5.html:36]: proper indent all JS code

File: Ashwin/JavaScript/09_Assignment/09_Assignment_6.html [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+            rel="stylesheet"
+            integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+            crossorigin="anonymous"
+        />
+        <script
+            src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+            crossorigin="anonymous"
+        ></script>
```
Comment (nileshp1122) [inline Ashwin/JavaScript/09_Assignment/09_Assignment_6.html]: proper indent script tag

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**